lambdas/api/get_dmps_downloads/src/__tests__/index.spec.ts

```diff
@@ -124,17 +124,18 @@ describe('handler', () => {
   });
 
   it('returns a 200 with the pre-signed URLs', async () => {
-    const expected = [
-      { fileName: 'testOne.json', url: 'http://example.com/test/1' },
-      { fileName: 'testTwo.json', url: 'http://example.com/test/2' },
-    ];
+    const expected = {}
+    expected['testOne.json'] = 'http://example.com/test/1';
+    expected['testTwo.json'] = 'http://example.com/test/2';
     mockGetExport.mockResolvedValue('Test');
     mockVerifyAPIGatewayLambdaAuthorizer.mockResolvedValue({ name: 'Tester' });
-    mockListObjects.mockResolvedValue([{ key: 'testOne' }, { key: 'testTwo' }]);
-    mockGetPresignedURL.mockResolvedValueOnce(expected[0]);
-    mockGetPresignedURL.mockResolvedValueOnce(expected[1]);
+    mockListObjects.mockResolvedValue([{ key: 'testOne.json' }, { key: 'testTwo.json' }]);
+    mockGetPresignedURL.mockResolvedValueOnce(expected['testOne.json']);
+    mockGetPresignedURL.mockResolvedValueOnce(expected['testTwo.json']);
     const response = await handler(mockEvent, mockContext, undefined);
 
+console.log(response.body)
```

Comment:
L137: Did you intentionally leave `console.log()` here?

Reply:
no, will remove that